src/uml/requestHandling.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+!include clean.skin
+"SQL Client" -> "Exasol Core" : Select Request
+activate "Exasol Core"
+"Exasol Core" -> "Exasol Core": parse
+"Exasol Core" -> "Virtual Schema Adapter": getCapabilities
+activate "Virtual Schema Adapter"
+"Exasol Core" <<-- "Virtual Schema Adapter": list of capabilities
+deactivate "Virtual Schema Adapter"
+"Exasol Core" -> "Exasol Core": rewrite to remove unsupported parts
+"Exasol Core" -> "Virtual Schema Adapter": pushdown("SELECT ...")
+activate "Virtual Schema Adapter"
+"Exasol Core" <<-- "Virtual Schema Adapter": "IMPORT FROM JDBC ..."
+deactivate "Virtual Schema Adapter"
+"Exasol Core" -> Importer: IMPORT FROM JDBC ...
+activate "Importer"
+"Exasol Core" <<-- Importer: rows
+deactivate "Importer"
```

Comment:
Please, add a few lines about Importer. It can be hard for a person who doesn't know about VS a lot to understand what is the importer

Reply:
And also mention that it's possible to avoid using importer if we use SELECT * FROM VALUES